docs/usage.rst

```diff
@@ -588,6 +588,120 @@ Legend:
 * Mode: The operational mode of the CPC: C=Classic, D=DPM
 * Type: The Prometheus metric type: G=Gauge, C=Counter
 
+Labels on exported metrics
+--------------------------
+
+The metrics exported to Prometheus are tagged with labels defined in the
+HMC credentials file (at the global level) or in the metric definition file
+(at the level of HMC metric groups and HMC metrics).
+
+This section describes what kinds of labels can be defined in these files. For
+the actual use of this capability, see the example metric definition file and
+the example HMC credentials file.
+
+Labels at the global level (defined in the HMC credentials file):
+
+.. code-block:: yaml
+
+    extra_labels:
+      - name: {label-name}
+        value: {label-value}
+
+Where:
+
+* ``{label-name}`` - Name of the label.
+* ``{label-value}`` - A :term:`Jinja2 expression` that is evaluated and used as
+  the value of the label.
+
+  The following variables are available for use in that Jinja2 expression:
+
+  - ``hmc_info`` - Dictionary with the result of the "Query API Version"
+    HMC operation. For details, see the description of that operation in the
+    :term:`HMC API` book. Some notable dictionary items are:
+
+    - ``hmc-name`` - Name of the HMC.
+    - ``hmc-version`` - Version of the HMC as a string (e.g. "2.16").
+
+Example (fragments from an HMC credentials file):
+
+.. code-block:: yaml
+
+    extra_labels:
+      - name: hmc
+        value: "{{ hmc_info['hmc-name'] }}"  # HMC name obtained from HMC
+      - name: dc
+        value: dal13                         # Literal value 'dal13'
+
+Labels at the HMC metric and HMC metric group level (defined in the metric
+definition file):
+
+.. code-block:: yaml
+
+    labels:
+      - name: {label-name}
+        value: {label-value}
+
+Where:
+
+* ``{label-name}`` - Name of the label.
+
+* ``{label-value}`` - A :term:`Jinja2 expression` that is evaluated and used as
+  the value of the label.
+
+  The following variables are available for use in that Jinja2 expression:
+
+  - ``resource_obj`` - The resource to which the metric applies, as a
+    :class:`zhmcclient.BaseResource` object. This can be used for example to
+    get the name of the resource or its parent or grand-parent resources,
+    or to get other propertes of the resource.
+
+  - ``metric_values`` - Dictionary of all metric values in the same metric group,
+    for the resource to which the metric applies. Key is the HMC metric name.
+    This variable is only present for HMC metrics based metrics, but not for
```

Comment:
HMC-based metrics?

Reply:
DONE - Made the term consistent with how we define them in section "Available metrics":
* HMC metric service based
* HMC resource property based